from geweb.exceptions import GewebError

class PointError(GewebError):
    pass

